docs/docs/pixabay-source-plugin-tutorial.md

```diff
@@ -96,7 +96,7 @@ to create the file using default options.
 
 ### Add dependencies
 
-You'll use a couple of modules from npm to add some helper functionality. Install them with:
+You'll use a couple of modules from `npm` to add some helper functionality. Install them with:
```

Comment:
Please revert changes related to putting npm into a code block.

Reply:
yes i can revert it - sure. 
i did it because a fews sentences above npm was in a code block - right under '### Create a `package.json` file'. 
what's the difference? or should we remove the ticks from the other npm sentence as well?

Reply:
npm shouldn't be in the code blocks, so I'd highly appreciate it if you could change these other occurrences in the file, too. Thanks! @codingCookie

Reply:
👍  removed all code-blocks regarding npm